pages/posts/content.json

```diff
@@ -1,5 +1,15 @@
 {
   "items": [
+    {
+      "id": "5",
+      "title": "5 - Wesh: Flight of a Byte",
+      "pubDate": "20/07/2023",
```

Comment:
At the bottom of previous blog posts, it says something like "Posted by Jeff on 20/07/2023". Should that be added automatically by this metadata? In any case, the current blog post does not have a "posted" date. Can you add one?